<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painoindeksilaskin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sukupuoli</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Paino (kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pituus (cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Painoindeksi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tulkinta</string>
    </property>
   </widget>
   <widget class="QLabel" name="sukupuoli">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nainen</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="paino_spinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> kg</string>
    </property>
    <property name="maximum">
     <number>299</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="pituus_spinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string> cm</string>
    </property>
    <property name="maximum">
     <number>299</number>
    </property>
   </widget>
   <widget class="QSlider" name="paino_slider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>371</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>299</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="pituus_slider">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>371</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximum">
     <number>299</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="bmi">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="tulkinta">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuPainoindeksilaskin">
    <property name="title">
     <string>Toimenpide</string>
    </property>
    <addaction name="actionAlkutila"/>
    <addaction name="actionLopeta"/>
   </widget>
   <widget class="QMenu" name="menuSukupuoli">
    <property name="title">
     <string>Sukupuoli</string>
    </property>
    <addaction name="actionNainen"/>
    <addaction name="actionMies"/>
   </widget>
   <addaction name="menuPainoindeksilaskin"/>
   <addaction name="menuSukupuoli"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAlkutila">
   <property name="text">
    <string>Alkutila</string>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionLopeta">
   <property name="text">
    <string>Lopeta</string>
   </property>
  </action>
  <action name="actionNainen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nainen</string>
   </property>
  </action>
  <action name="actionMies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mies</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pituus_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pituus_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pituus_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pituus_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paino_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>paino_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paino_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>paino_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
